index.html

```diff
@@ -11,22 +11,200 @@
 <body>
     <header class="navbar">
         <a class="welcome">Welcome</a>
-        <nav><img class="menu" alt="button" src="Enabled.png" ></nav>
+        <nav><img class="menu" alt="button" src="images/Enabled.png" ></nav>
     </header>
     <section class="main">
-        <h1>Hey there. I’m Amalia</h1>
-        <h2>I’m a software developer</h2>
-        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        <h1 class="amalia">Hey there. I’m Amalia</h1>
+        <h2 class="developer">I’m a software developer</h2>
+        <p class="contact"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
             don’t hestiate to contact me.</p>
-    
+            
     <ul class="icons">
-        <li><a href="www.twitter.com" ><img src="twitter.png" alt="twitter"></a></li>
-        <li><a href="www.linkedin.com"><img src="Linkedin.png" alt="Linkedin"></a></li>
-        <li><a href="#"><img src="medium.png" alt="medium"></a></li>
-        <li><a href="#"><img src="github.png" alt="github"></a></li>
-        <li><a href="#"><img src="angellist.png" alt="angellist"></a></li>    
+        <li><a href="www.twitter.com"><img src="images/twitter.png" alt="twitter"></a></li>
+        <li><a href="www.linkedin.com"><img src="images/Linkedin.png" alt="Linkedin"></a></li>
+        <li><a href="#"><img src="images/medium.png" alt="medium"></a></li>
+        <li><a href="#"><img src="images/github.png" alt="github"></a></li>
+        <li><a href="#"><img src="images/angellist.png" alt="angellist"></a></li>    
     </ul>  
     </section>
+
+    <header class="title">
```

Comment:
- [ ] The projects cards are not the same as the Figma design, please consider adjusting the styles for the cards

|Your design|Figma design|
|-|-|
|![Screenshot_20220609_163533](https://user-images.githubusercontent.com/25227022/172887345-e250db16-fd34-4ec6-a50f-ce1bcb6d39a5.png)|![Screenshot_20220609_163545](https://user-images.githubusercontent.com/25227022/172887308-9ed13240-a196-4793-b20b-e271ccf9bd60.png)|